src/combinatorics/configuration.lean

```diff
@@ -301,4 +299,37 @@ let this := @has_lines.has_points (dual L) (dual P) _ _ _ _ h.symm in
 { mk_line := this.mk_point,
   mk_line_ax := this.mk_point_ax }
 
+variables (P L)
+
+/-- A nondegenerate configuration satisfying `has_points`, `has_lines`, and an additional
+  nondegeneracy condition. -/
```

Comment:
Could you please be a bit more specific in this docstring? E.g., it doesn't mention the words "projective plane" at all.